charts/base/templates/keda.yaml

```diff
@@ -11,12 +11,16 @@ spec:
   minReplicaCount:  {{ .Values.autoscaling.minReplicas }}
   maxReplicaCount: {{ .Values.autoscaling.maxReplicas }}
   triggers:
-    - type: {{ .Values.autoscaling.trigger.type }}
+    {{- range .Values.autoscaling.trigger }}
```

Comment:
lets support both trigger and triggers, both with single/list ability, with backward compatibility, check config, ingress, multi-containers options we have with similar ways implemented